common/buildcraft/transport/plug/PluggableFacade.java

```diff
@@ -108,6 +113,11 @@ public float getExplosionResistance(@Nullable Entity exploder, Explosion explosi
         return states.phasedStates[activeState].stateInfo.state.getBlock().getExplosionResistance(exploder);
     }
 
+    @Override
+    public BlockFaceShape getBlockFaceShape() {
```

Comment:
travis is failing on this override, you probably added this method in IFacade but that is part of the api (in another repo), could you make a PR there as well for this?

Reply:
Done.